package.json

```diff
@@ -20,21 +20,25 @@
   "homepage": "https://react-countup.now.sh",
   "main": "build",
   "files": [
-    "build",
-    "index.d.ts"
+    "build"
   ],
-  "typings": "index.d.ts",
+  "typings": "build/index.d.ts",
   "scripts": {
-    "build": "rollup -c",
+    "build": "rollup -c && tsc --emitDeclarationOnly --noEmit false --project src/tsconfig.json --outDir build",
     "prepublishOnly": "yarn build",
     "test": "jest"
   },
   "peerDependencies": {
     "react": ">= 16.3.0"
   },
   "dependencies": {
+    "@babel/preset-typescript": "^7.14.5",
+    "@rollup/plugin-node-resolve": "^13.0.2",
+    "@rollup/plugin-typescript": "^8.2.3",
+    "@types/warning": "^3.0.0",
     "countup.js": "^2.0.7",
     "prop-types": "^15.7.2",
```

Comment:
i suppose that we can remove `prop-types` from packages and `rollback` config

Reply:
derp, yes I've now cleaned up package.json to put deps in the right place and remove `@rollup/plugin-typescript` and `prop-types`.